Components/FlightControl/WatchdogTask.cpp

```diff
@@ -0,0 +1,120 @@
+/**
+ ******************************************************************************
+ * File Name          : WatchdogTask.cpp
+ * Description        : Primary Watchdog task, default task for the system.
+ ******************************************************************************
+*/
+#include "GPIO.hpp"
+#include "SystemDefines.hpp"
+#include "Timer.hpp"
+#include "WatchdogTask.hpp"
+
+
+/**
+ * @brief Constructor for WatchdogTask
+ */
+WatchdogTask::WatchdogTask() : Task(WATCHDOG_TASK_QUEUE_DEPTH_OBJS)
+{
+}
+
+/**
+ * @brief Initialize the WatchdogTask
+ */
+void WatchdogTask::InitTask()
+{
+    // Make sure the task is not already initialized
+    SOAR_ASSERT(rtTaskHandle == nullptr, "Cannot initialize watchdog task twice");
+
+    BaseType_t rtValue =
+        xTaskCreate((TaskFunction_t)WatchdogTask::RunTask,
+            (const char*)"WatchdogTask",
+            (uint16_t)WATCHDOG_TASK_STACK_DEPTH_WORDS,
+            (void*)this,
+            (UBaseType_t)WATCHDOG_TASK_RTOS_PRIORITY,
+            (TaskHandle_t*)&rtTaskHandle);
+
+    SOAR_ASSERT(rtValue == pdPASS, "WatchdogTask::InitTask() - xTaskCreate() failed");
+}
+
+/**
+ * @brief This function is called if the heartbeat timer expires and it sends a command to ABORT the system
+ * @param Automatically passes is timer handle for callback, should not be used
+ */
+void WatchdogTask::HeartbeatFailureCallback(TimerHandle_t rtTimerHandle)
+{
+
+    SOAR_PRINT("The system lost its heartbeat and had to reset!!!\n");
+//    SOAR_ASSERT(false ,"The system lost its heartbeat and had to reset!!! \n");
```

Comment:
Change this to `"System lost radio heartbeat: attempting to ABORT\n"`

Remove the commented out line

Reply:
ahhhh!!! I removed that line but it wasn't committed